unfoldingWord/translationCore
# Support for new new UGNT v0.2 and lexicon

#### This pull request addresses:

Replaces Greek UGNT with v0.2 and updated lexicon.

#### How to test this pull request:

- Check out the same branch on Core
- Delete your ~/translationCore/resources folder so that launching the app populates the resources folder
- Load a Titus project, and tW, click on the Greek words in the scripture pane to get the Lexicon entries.
- Load a non-Titus project and tW, verify that the app doesn't crash but displays placeholder text.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2722)
